server/templates/details.html.tmpl

```diff
@@ -56,6 +56,24 @@
 <li data-status="{{$s}}">
   <div class="bg-white p-2 shadow-sm max-w-lg status-stripe {{$s}}">
     {{template "result-details" .}}
+    {{if (or .PredicateResults (hasRequires .Requires))}}
+    <details class="bg-light-gray5 p-2 mt-2 text-sm">
+      <summary class="text-dark-gray3 font-semibold">Details</summary>
+      <div class="pt-2"></div>
+      <div class="border-t border-light-gray3">
```

Comment:
I think you should be able to combine these `div`s by adding `mt-2` to the `div` with the border. I like to avoid empty `div` when I can, and it should be possible here.